### PR TITLE
4.next to 5.x merge

### PR DESCRIPTION
**Pull request by othercorey**
This is a clean merge of the mess in 4.next into 5.x.

I manually diffed every file that changed and used the original file from 5.x branch anywhere a file was changed only by https://github.com/cakephp/docs/commit/54346087083a4ea10a7301dce7c63a4beae5c5e3.

There were a few mixed files so I did my best to resolve the expected changes.

### CODE REVIEW

File: en/core-libraries/internationalization-and-localization.rst [hunk 1]
```diff
@@ -103,7 +103,7 @@ Extract Pot Files with I18n Shell
 ---------------------------------
 
 To create the pot files from `__()` and other internationalized types of
-messages that can be found in the application code, you can use the i18n command.
+messages that can be found in the application code, you can use the i18n shell.
```
Comment (ADmad) [inline en/core-libraries/internationalization-and-localization.rst]: Should be `command` not `shell` and "i18n" should be in back ticks.

File: en/core-libraries/time.rst [hunk 2]
```diff
@@ -497,4 +497,4 @@ and parse localized datetime strings. See the :ref:`parsing-localized-dates`.
 Supported Timezones
 ===================
 
-CakePHP supports all valid PHP timezones. For a list of supported timezones, `see this page <https://php.net/manual/en/timezones.php>`_.
+CakePHP supports all valid PHP timezones. For a list of supported timezones, `see this page <http://php.net/manual/en/timezones.php>`_.
```
Comment (ADmad) [inline en/core-libraries/time.rst]: With "s"

File: en/development/errors.rst [hunk 3]
```diff
@@ -64,7 +65,7 @@ deprecated. We also recommend this system for use in your plugins and
 application code when useful. You can trigger deprecation warnings with
 ``deprecationWarning()``::
 
-    deprecationWarning('5.0', 'The example() method is deprecated. Use getExample() instead.');
+    deprecationWarning('The example() method is deprecated. Use getExample() instead.');
```
Comment (ADmad) [inline en/development/errors.rst]: Version no. is needed as 1st argument in 5.x.

File: fr/appendices/4-0-upgrade-guide.rst [hunk 4]
```diff
@@ -49,9 +49,9 @@ il existe un outil CLI de mise à niveau:
 .. code-block:: console
 
     # Installe l'outil d'upgrade
-    git clone git://github.com/cakephp/upgrade
+    git clone https://github.com/cakephp/upgrade
     cd upgrade
-    git checkout master
+    git checkout 4.x
```
Comment (ADmad) [inline fr/appendices/4-0-upgrade-guide.rst:54]: This adds back 4.x migration guides which I already dropped on `5.x` branch.

File: en/development/errors.rst (hunk 3 above)
Comment (othercorey) [reply en/development/errors.rst]: Looks like a couple reverts didnt take.

File: en/core-libraries/internationalization-and-localization.rst (hunk 1 above)
Comment (othercorey) [reply en/core-libraries/internationalization-and-localization.rst]: Reverted file to 5.x version.

File: en/core-libraries/time.rst (hunk 2 above)
Comment (othercorey) [reply en/core-libraries/time.rst]: Reverted file.

File: en/development/errors.rst (hunk 3 above)
Comment (othercorey) [reply en/development/errors.rst]: Fixed this and removed references to shell/task exceptions. The remaining changes are from 4.x.

**Review by ADmad — APPROVED** [review]

**Pull request merged**